### DISCUSSION TITLE
阿里云盘报错 Post "https://api.nn.ci/alist/ali_open/token": dial tcp 45.66.130.152:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.

### DISCUSSION THREAD

**Discussion by YuanKun666** [AlistGo/alist#4577] [General]
![image](https://github.com/alist-org/alist/assets/76767255/150eab73-dd0b-4b0f-a0c3-fc53b07108ae)

**Comment by ghost** [comment]
我也遇到了这个问题 不过已经解决了
将配置里的"Oauth token url"项中的 https://api.nn.ci/ 改成 https://api-cf.nn.ci/ 或者 https://api.xhofe.top/ 试试。

**Reply by cmj2023** [reply]
我是把 Oauth令牌链接 项的 https换成了http

**Reply by heqisong** [reply]
多谢分享  按照你的方式处理，已经OK拉

**Reply by JackWoo925** [reply]
Thanks a lot for you help!

**Comment by rexhang** [comment]
感谢

**Reply by FEIXIANGDEZHU666** [reply]
感谢  修好了

**Comment by Huanxige52** [comment]
**我把 Oauth令牌链接 项的 https换成了http  就好了。阿里云open**

**Comment by JackWoo925** [comment]
> 我也遇到了这个问题 不过已经解决了 将配置里的"Oauth token url"项中的 https://api.nn.ci/ 改成 https://api-cf.nn.ci/ 或者 https://api.xhofe.top/ 试试。
